test/gurka/test_match.cc

```diff
@@ -168,7 +168,7 @@ class TrafficBasedTest : public ::testing::Test {
                                 {"mjolnir.shortcuts", "false"},
                                 {"mjolnir.timezone", VALHALLA_BUILD_DIR "test/data/tz.sqlite"},
                             });
-    map.config.put("mjolnir.traffic_extract", "test/data/algorithm_selection/traffic.tar");
+    map.config.put("mjolnir.traffic_extract", "test/data/traffic_test/traffic.tar");
```

Comment:
The path here should be `test/data/match_timedep/traffic.tar`

Gurka creates `test/data/match_timedep` and puts `map.pbf` in there - the directory will exist to also put your `traffic.tar` file in.

Reply:
updated it. I'm keeping the change to create dir if it doesn't exist, unless you think its better to throw an exception. let me know